crates/rust-analyzer/src/config.rs

```diff
@@ -215,6 +215,9 @@ config_data! {
         /// Join lines merges consecutive declaration and initialization of an assignment.
         joinLines_joinAssignments: bool = "true",
 
+        /// Pre-fill caches on project load.
+        load_prefillCaches: bool = "true",
```

Comment:
Bikeshed this.

Reply:
`rust-analyzer.cache.prefill`?

Reply:
This is often called cache `warmup`.

Reply:
`rust-analyzer.cache.warmup` would also be good.